docs/user-guide.md

```diff
@@ -56,6 +56,8 @@ Usage of kube-router:
       --disable-source-dest-check                     Disable the source-dest-check attribute for AWS EC2 instances. When this option is false, it must be set some other way. (default true)
       --enable-cni                                    Enable CNI plugin. Disable if you want to use kube-router features alongside another CNI plugin. (default true)
       --enable-ibgp                                   Enables peering with nodes with the same ASN, if disabled will only peer with external BGP peers (default true)
+      --enable-ipv4                                   Enables IPv4 support (default true)
+      --enable-ipv6                                   Enables IPv6 support (default true)
```

Comment:
Is this needed?

Reply:
Yeah, I'm adding `EnableIPv4` and `EnableIPv6` to pkg/options/options.go and they are going to show up as CLI arguments.

Reply:
I mean, I know that doesn't matter for the way we use kube-router in k3s, but I would like to eventually upstream this.